src/components/ModalWrapper.tsx

```diff
@@ -6,6 +6,7 @@ interface ModalProps {
   header: string
   open: boolean
   content: JSX.Element
+  styleOverrides: { [key: string]: string | number }
```

Comment:
Adding this as a non-optional prop is causing a type check failure in `CommandBlocklistView.tsx`.